Dockerfile

```diff
@@ -4,12 +4,16 @@ FROM golang:1.13.1-alpine AS base
 ENV GO111MODULE on
 ENV GOPROXY https://proxy.golang.org
 ENV CGO_ENABLED 0
+RUN apk --no-cache add bash curl
```

Comment:
do you need a `api add --update --no-cache` in here?

Reply:
nice catch, thanks!